internal/saucecloud/cloud.go

```diff
@@ -473,25 +470,32 @@ func (r *CloudRunner) remoteArchiveFiles(project interface{}, files []string, sa
 		return "", err
 	}
 
-	zipName, err := r.archiveFiles("app", tempDir, ".", files, matcher)
+	zipName, err := zip.ArchiveFiles("app", tempDir, ".", files, matcher)
 	if err != nil {
 		return "", err
 	}
 	archives[zipName] = projectUpload
 
-	configZip, err := r.archiveRunnerConfig(project, tempDir)
+	configZip, err := zip.ArchiveRunnerConfig(project, tempDir)
 	if err != nil {
 		return "", err
 	}
 	archives[configZip] = runnerConfigUpload
 
 	var uris []string
-	for k, v := range archives {
-		uri, err := r.uploadProject(k, "", v, dryRun)
-		if err != nil {
-			return "", err
+	// keep uploaded file order
```

Comment:
The current order is random. So the values of `app` and `otherApps` are also assigned randomly https://github.com/saucelabs/saucectl/blob/main/internal/saucecloud/cypress.go#L114
To update the runner config URI, I need to set them in order. I'll also add it in comments

Reply:
https://github.com/saucelabs/saucectl/blob/main/internal/saucecloud/cypress.go#L114-L115 perhaps a code smell that we should improve here. Make it more explicit rather than keeping the implicit knowledge of the order. The API accepted a list of URIs, because the order did not matter. The order should still not matter.